src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -233,7 +233,7 @@ class DataPointsResource[F[_]](val requestSession: RequestSession[F])
       id: Long,
       inclusiveStart: Instant,
       exclusiveEnd: Instant,
-      limit: Option[Int] = None
+      limit: Option[Int] = Some(100000)
```

Comment:
Create a Constants object and put this in there, and give this a name. We should avoid magic numbers.
This change makes it look like we're not setting appropriate limits for other resource types either, just using the default values? :/